src/Logic.hs

```diff
@@ -369,16 +369,16 @@ handlePullRequestEdited prId newTitle newBaseBranch state =
     -- Do nothing if the pull request is not present.
     Nothing -> pure state
 
--- Internal result type for parsing a merge command, which allows the
--- consumer of `parseMergeCommand` to inspect the reason why a message
--- was considered invalid.
+-- | Internal result type for parsing a merge command, which allows the
+-- | consumer of `parseMergeCommand` to inspect the reason why a message
+-- | was considered invalid.
```

Comment:
These pipes should only go in front on the first comment. See https://haskell-haddock.readthedocs.io/en/latest/markup.html.

Reply:
Thanks! Fixed :)